### DISCUSSION TITLE
MS SQL Support

### DISCUSSION THREAD

**Discussion by tanj** [SeaQL/sea-orm#578] [General]
It appears as though sqlx [supports mssql databases](https://github.com/launchbadge/sqlx/tree/master/sqlx-core/src/mssql). I'm wondering since that is the case and this projects depends on sqlx; what it would take to support mssql in this project?

**Comment by billy1624** [comment]
Hey @tanj, good question! We are planning to support MSSQL in near future. Basically we need to implement the MSSQL SQL dialect in SeaQuery and support schema discovery of MSSQL in SeaSchema before fully support it in SeaORM.

Related Issues:
- SeaQL/sea-schema#48
